src/components/Modals/Receive/Receive.tsx

```diff
@@ -18,14 +18,12 @@ const Receive = ({ asset, accountId }: ReceivePropsType) => {
   const { receive } = useModal()
   const { close, isOpen } = receive
 
-  if (!asset) return null
```

Comment:
Derp unsafe safety while refactoring the API of `<ReceiveRouter />` to receive an `assetId` instead of an `asset` - undefined `assetId` (`asset` previously) is perfectly valid.